fastlane/Fastfile

```diff
@@ -66,7 +66,13 @@ ALL_LOCALES = [
   { glotpress: "kmr", android: "kmr", promo_config: false},
 ].freeze
 
-RELEASE_NOTES_LOCALES = ALL_LOCALES.reject { |h| h[:google_play].nil? }.map { |h| [h[:glotpress], h[:google_play]] }
+RELEASE_NOTES_LOCALES = ALL_LOCALES
+  .reject { |h| h[:google_play].nil? }
+  .map { |h| [h[:glotpress], h[:google_play]] }
+JP_RELEASE_NOTES_LOCALES = ALL_LOCALES
+  .reject { |h| h[:google_play].nil? }
+  .select { |h| %w[ar de-DE es-ES fr-FR iw-IL id it-IT ja-JP ko-KR nl-NL pt-BR ru-RU sv-SE tr-TR zh-CN zh-TW].include?(h[:google_play]) }
+  .map { |h| [h[:glotpress], h[:google_play]] }
```

Comment:
Side note: I already have plans/ideas to make that mess with `ALL_LOCALES` better (TL;DR move the list into the release-toolkit, which should also provide convenience methods to build the subsets of locales from just the google_play or glotpress (or whatever reference we want to use) locale codes). This should allow us to clean up lines 10-67 and make the above lines 69-75 simpler and easier to read to.

But all that is for another PR

Reply:
Nice! You can ignore my comment above. I am just keeping it for full disclosure.